data/hex-base64-encoding.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @title Hex and Base64 Encoding
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ *
+ * There are a few cases where it would be practical to encode
+ * and decode between different string and array buffer formats.
+ * The Deno standard library makes this easy.
+ */
+
+// The standard library provides hex and base64 encoding and decoding utilities
+import * as base64 from "https://deno.land/std@0.171.0/encoding/base64.ts";
```

Comment:
The other examples use `std@0.168.0`. Until `denobyexample` standardizes their std example, it would be best to use the other std versions.

Reply:
The other examples were updated from older versions of the std (I believe it was actually one of my PRs). I believe the policy is to "just use the latest available version at the time and we will update the dependencies at some point later to keep them in sync".